sections/builder.js

```diff
@@ -1561,6 +1561,40 @@ export default [
       }
     ]
   },
+  {
+    type: "method",
+    method: "skipLocked",
+    example: ".skipLocked()",
+    description: "PostgreSQL only. This method can be used after a lock mode has been specified with either forUpdate or forShare, and will cause the query to skip any locked rows, returning an empty set if none are available.",
```

Comment:
@kukiric Probably this needs to be updated to mention MySQL 8.0 as well?

Reply:
Correct, both methods need to be updated after the last few changes.